trunk/SUAVE/Methods/Performance/estimate_landing_field_length.py

```diff
@@ -115,8 +115,7 @@ def estimate_landing_field_length(vehicle,analyses,airport):
         landing_constants[0] = 250.
         landing_constants[1] =   0.
         landing_constants[2] =  2.485  / sea_level_gravity  # Two-wheels truck : [ (1.56 / 0.40 + 1.07) / (2*sea_level_gravity) ]
-##        landing_constants[2] =   2.9725 / sea_level_gravity  # Four-wheels truck: [ (1.56 / 0.32 + 1.07) / (2*sea_level_gravity) ]
```

Comment:
this shouldn't be deleted.

Make an option for four wheel trucks

Reply:
1) The line has been commented out for the longest time 2) It will overwrite what is there before for two-wheel trucks. How do you think I should proceed

Reply:
not your problem at this time, but it should query the landing gear and see how many wheels it has:https://github.com/suavecode/SUAVE/blob/develop/trunk/SUAVE/Components/Landing_Gear/Main_Landing_Gear.py#L52

For now, just don't delete it so we can go back

Reply:
Delete reverted